eval/mt_bench_pipeline.py

```diff
@@ -121,7 +120,7 @@ def run_mt_bench(
     with open(mt_bench_output.path, 'w') as f:
         json.dump(mt_bench_data, f, indent=4)
 
-@component(base_image=TOOLBOX_IMAGE)
+@component(base_image=PYTHON_IMAGE)
```

Comment:
Is the `PYTHON_IMAGE` imported in this file?

Reply:
I believe so, it would fail otherwise, no? I think I remember adding it.... looking...